jetty-websocket/websocket-jetty-client/src/main/config/modules/websocket-jetty-client.mod

```diff
@@ -0,0 +1,24 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Expose the Jetty WebSocket Client classes to deployed web applications.
+
+[tags]
+websocket
+
+[depend]
+client
+annotations
+
```

Comment:
Should this module depend on the jetty server websocket impl?    Once we expose the API aren't we committed to supplying the server side?

Reply:
We're not committed to supplying the server side by having the client. The websocket client can be used without any server dependencies.